samples/sensor/stm32_vbat_sensor/README.rst

```diff
@@ -47,4 +47,4 @@ Sample Output
 
 .. code-block:: console
 
-Current Vbat voltage: 3.04 V
+    Current Vbat voltage: 3.04 V
```

Comment:
This fix isn't quite right. You've added one space too many

Reply:
Thanks for the correction. I have force pushed the new change.